components/application-registry/internal/metadata/applications/util.go

```diff
@@ -172,6 +197,21 @@ func getServiceIndex(id string, app *v1alpha1.Application) int {
 	return -1
 }
 
+func normalizeName(displayName string) string {
```

Comment:
Is the normalization function (how it should work) documented somewhere ?

Reply:
I'm afraid not